IsTheMartin/mrtnmrls
# Post pages

## Summary

Post pages were created based on language to contain a list of posts.

Also, post page was modified to include new footer and header, some elements were removed such as navigation component. Thanks component was added in post layout file to contain RSS and posts list as links

## Evidence

Please include some screenshot if apply 

<img width="678" alt="image" src="https://github.com/IsTheMartin/mrtnmrls/assets/38388605/8b66887c-4c95-4d2e-b2f4-c7bffa410fd9">

## Concern Information

To test please follow the next steps:

1. Scroll to footer section in index
2. Click "posts" link
3. Observe
4. Click on any post
5. Scroll to bottom of the page
6. Observe Footer and section before Footer

## Issue ticket

- resolves #38 

## Check list

- [ x ] I did add reviewers to this PR
